Dockerfile

```diff
@@ -148,9 +150,11 @@ LABEL maintainer="Ilios Project Team <support@iliosproject.org>"
 
 # semi-colon seperates list of github users that can SSH in
 ENV GITHUB_ACCOUNT_SSH_USERS=''
+# set default LOCK_DSN for opensearch
```

Comment:
This doesn't have anything to do with open search, it's a default [provided by Symfony](https://symfony.com/doc/current/lock.html)

Reply:
removed!